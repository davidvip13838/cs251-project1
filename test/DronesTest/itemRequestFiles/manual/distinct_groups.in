6 2
0 3 4 12 17 19
