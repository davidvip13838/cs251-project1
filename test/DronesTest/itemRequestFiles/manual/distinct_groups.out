2 6
1 10
0 12
3 16
5 21
4 25
